src/applications/claims-status/tests/components/OverviewPage.unit.spec.jsx

```diff
@@ -14,63 +15,23 @@ const params = { id: 1 };
 describe('<OverviewPage>', () => {
   const store = createStore(() => ({}));
 
-  context('DDL feature flag is enabled', () => {
```

Comment:
Removing this because we will be removing the <ClaimsDecision> component from the OverviewPage container soon anyways. @pmclaren19 said it was ok